src/friendly-iframe-embed.js

```diff
@@ -444,6 +444,16 @@ export class FriendlyIframeEmbed {
       this.visibilityObservable_.fire(this.visible_);
     }
   }
+
+  setCssAnimPlayingState(running) {
+    if (platformFor(this.win).isIos()) {
+      this.win.document.documentElement.classList.toggle(
```

Comment:
So, this is not the full picture and I'd like to start from the "beginning" here. Here are some aspects that are not clear to me and need to be confirmed first:

1. How do we block animations. I suggested before that we could set `* {animation-name: none}`. However, I have two concerns:
 a. Except for FIE (e.g. in-a-box), we have no way to set this style before AMP Runtime is downloaded. That means that we have a risk condition and animation can start running automatically before AMP Runtime is installed.
 b. What's worse in this case, is that with some animation parameters (`animation-fill-mode`), we may get in a place where the state of DOM is far away from the initial state after the reset. This will yield the corrupted UI and incorrect animation when we do allow it.
 c. Normally we always want to wait for animated elements to be loaded before we start the animation. CSS animations do not naturally give us that option.
2. How do we unblock animations. I think we are all in agreement it'd be nice to be able to do so by adding or removing a class.
3. Finally, we still need some research on pausing and iOS. My hope that a non-scrollable iframe embed will be ok to use `animation-play-state: paused`.

We need to research these topics in great detail and prototype. Then we need to make a decision for the format.